### PR TITLE
Cord blood mtd

### PR DESCRIPTION
**Pull request by drighelli**
adding metadata/colData for cord_blood dataset in CITEseq.

providing rda object through slack.

### CODE REVIEW

File: R/CITEseq.R [hunk 1]
```diff
@@ -192,7 +215,9 @@ CITEseq <- function(DataType=c("cord_blood", "peripheral_blood"), modes="*",
             ## Add here other CITE-seq datasets based on DataType identifier
             { stop("Unrecognized CITE-seq dataset name: ", DataType) }
         )
-        
+        if (filtered==TRUE) {
```
Comment (LiNk-NY) [inline R/CITEseq.R]: This should be `if (filtered)`

**Review by LiNk-NY — APPROVED** [review]

**Comment by LiNk-NY** [comment]
Thank Dario @drighelli, it looks good. The merge should wait until we have the resources uploaded to `ExperimentHub`.

File: R/CITEseq.R (hunk 1 above)
Comment (drighelli) [reply R/CITEseq.R]: right! will be fixed in the next commit!

**Comment by drighelli** [comment]
I can see that the latest commits on peripheral_blood dataset are already tracked in this PR.

**Comment by LiNk-NY** [comment]
Hi Dario, @drighelli 
I don't see an entry in the metadata for the `peripheral_blood/v1.0.0/scRNAseq_coldata.rda`. Can you add this?

**Review by LiNk-NY — CHANGES_REQUESTED** [review]
Please include an additional entry in the metadata for the `peripheral_blood` colData Rda file. Everything else looks good. Thanks

**Comment by drighelli** [comment]
ahah sorry I edited the file without pushing it. 

check it now please! 

Thanks for the review!

**Review by LiNk-NY — APPROVED** [review]

**Comment by LiNk-NY** [comment]
I'm still waiting on the updated EH database but the changes have been merged.

**Pull request merged**